controller/lib/core/blower_pid.cpp

```diff
@@ -98,19 +89,19 @@ void blower_pid_execute(const BlowerSystemState &desired_state,
 
   Pressure cur_pressure = get_patient_pressure();
 
-  Setpoint = desired_state.setpoint_pressure.kPa();
-  Input = cur_pressure.kPa();
-  myPID.Compute();
+  float output;
+  myPID.Compute(/*input=*/cur_pressure.kPa(),
+                /*setpoint=*/desired_state.setpoint_pressure.kPa(), &output);
 
   // If the blower is not enabled, immediately shut down the fan.  But for
   // consistency, we still run the PID iteration above.
   if (!desired_state.blower_enabled) {
-    Output = 0;
+    output = 0;
   }
-  Hal.analogWrite(PwmPin::BLOWER, static_cast<int>(Output));
+  Hal.analogWrite(PwmPin::BLOWER, static_cast<int>(output));
 
   // fan_power is in range [0, 1].
-  *fan_power = stl::min(stl::max(Output, 0.f), 255.f) / 255.f;
+  *fan_power = stl::min(stl::max(output, 0.f), 255.f) / 255.f;
```

Comment:
Again, not your PR, but I see the pid lib uses stl::clamp to achieve the first part of this. Does it make any difference at runtime?

Reply:
We actually don't need the clamping here, as the PID does it for us.